### PR TITLE
#1388 Strip View Widget - Phase 1

### PR DESCRIPTION
**Pull request by drakkein**
Realated to #1388

### CODE REVIEW

**Comment by drakkein** [comment]
I've created test bin with CSS proposed by @falinebambi 
https://jsbin.com/letewenuwo/edit?html,css,output

This is not what we wanted to achieve. I'll try to set some media queries.

**Comment by drakkein** [comment]
There are some visual regression issues, due of background change. I think we should set bg for storybook to make it pass. @desfero

File: app/components/eto/EtoWidgetView.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    return (
+      <Col xs={12}>
+        <div className="mb-3">
+          <EtoOverviewStatus
+            preMoneyValuationEur={eto.preMoneyValuationEur}
+            existingCompanyShares={eto.existingCompanyShares}
+            equityTokensPerShare={eto.equityTokensPerShare}
+            minimumNewSharesToIssue={eto.minimumNewSharesToIssue}
+            contract={eto.contract!}
```
Comment (Moejoe90) [inline app/components/eto/EtoWidgetView.tsx]: Is contract address optional in the data structure?
If so i would investigate if the data structure is not precise.
Or add a fallback value if it's undefined, or change the interface of the component

File: app/components/eto/EtoWidgetView.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  appConnect<IStateProps, IDispatchProps, IRouterParams>({
+    stateToProps: (state, props) => ({
+      userType: selectUserType(state.auth),
+      eto: selectEtoWithCompanyAndContractById(state, props.etoId),
+    }),
+  }),
+  onEnterAction({
+    actionCreator: (dispatch, props) => {
+      dispatch(actions.publicEtos.loadEto(props.etoId));
+      dispatch(actions.wallet.loadWalletData());
```
Comment (Moejoe90) [inline app/components/eto/EtoWidgetView.tsx]: Why do you need this action?
Does the component use wallet data of any sort?

File: app/components/eto/EtoWidgetView.tsx (hunk 1 above)
Comment (drakkein) [reply app/components/eto/EtoWidgetView.tsx]: In props interface contact could be undefinde, so this non-null assertion is unnecessary.

File: app/components/eto/EtoWidgetView.tsx (hunk 2 above)
Comment (drakkein) [reply app/components/eto/EtoWidgetView.tsx]: This is leftover after commit #39ee1ffbf2e5dbd0bce154e161f909472f1607fb where wallet was required in ETOOverviewStatus. Good catch.

**Pull request merged**